judgments/utils/search_request_to_parameters.py

```diff
@@ -38,71 +37,70 @@ def search_request_to_parameters(request: HttpRequest, return_page_if_invalid="p
     with the submission (i.e. Month > 12)
     """
     if not form.is_valid():
-        # If the form has errors, return it for rendering!
-        return TemplateResponse(request, return_page_if_invalid, {"form": form})
```

Comment:
The caller for the search page already checks the validity before the call. This isn't a sensible type to return here for the atom feed.

Reply:
The rest of the changes in this commit are just unindentation from making it an `if - return` not `if - return - else`